### ISSUE TITLE
VNC to MAC machine failes with Incompatible Version error

### ISSUE THREAD

**Issue by soumyathampi** [aerokube/selenoid-ui#457]
Hi,

I am running selenoid on mac machine using the command
./cm selenoid start --vnc -d
./cm selenoid-ui start -d

But while lauching the browser on selenoid ui the vnc gets disconnected with following error
failed when connecting: Security negotiation failed on authentication scheme (reason: Incompatible Version.)
_fail @ rfb.js:668

**Comment by vania-pooh** [comment]
@soumyathampi this is actually a bug in MacOS VNC implementation not following the standard. Related issue https://github.com/novnc/noVNC/issues/1464.

**Comment by soumyathampi** [comment]
@vania-pooh do you mean the issue is with mac default vnc server?
Is there any other mac vnc servers which will work?

**Comment by vania-pooh** [comment]
@soumyathampi yes, the issue is with Mac built-in server. We don't know about other VNC servers on Mac.
